test/Neuroglia.UnitTests/Cases/Serialization/YamlDotNetSerializerTests.cs

```diff
@@ -78,8 +78,8 @@ public async Task SerializeAndDeserialize_JTokenTypes_ShouldWork(JToken token, s
             new object[] { JObject.FromObject(new { }), "{}" },
```

Comment:
at line 73 better to use Environment.NewLine instead of "\r\n"

Reply:
I just trimmed both sides to ignore trailing spaces/tabs/returns